fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biomes/MixinVanillaLayeredBiomeSource.java

```diff
@@ -46,10 +46,4 @@
 	private static void cinit(CallbackInfo info){
 		biomes = new HashSet<>(biomes);
```

Comment:
Potentially unrelated comment: Are we sure this doesn't wipe existing vanilla biomes?

Reply:
how come?

Reply:
It's replacing the hash set with a new instance, so if this runs after the vanilla set was initialized it could be bad.